### DISCUSSION TITLE
Photon service - DSGVO/RGPD compliance

### DISCUSSION THREAD

**Discussion by joerggross** [komoot/photon#539] [Q&A]
Hi,

we think of using the photon-service hosted by Komoot in a small Web-App to validate customer addresses. 
We have a question concerning DSGVO/RGPD compliance using this service: Will the requested address data only be processed in memory and never stored to a persitent storage - including logfiles?

Thanks in advance for answering this question and for providing this great service ;-)

Best Regards,

Jörg Groß

**Comment by lonvia** [comment]
photon.komoot.io stores logs of the requests for a limited time to the extend as it is necessary to ensure the operation of the service. This data is not used in any other capacity or shared with any third party.                                                                                    
                                                                                                  
Please note that the service is provided as is and for demo purposes only. For production setups, in particular when handling sensitive data, you should always set up your own photon instance. Have a look at our data dumps which make the setup fairly easy.
